db/migrate/20170313192741_add_themes.rb

```diff
@@ -73,6 +73,6 @@ def up
   end
 
   def down
-    raise IrriversibleMigration
+    raise IrreversibleMigration
```

Comment:
could be retained

Reply:
Afaict, this raise failed because the class didn't exist, which would trigger a different exception  https://api.rubyonrails.org/classes/ActiveRecord/IrreversibleMigration.html